Commands/Lists/RemoveListItem.cs

```diff
@@ -20,7 +20,7 @@ public class RemoveListItem : SPOWebCmdlet
         [Parameter(Mandatory = true, HelpMessage = "The ID of the listitem, or actual ListItem object")]
         public ListItemPipeBind Identity;
 
-        [Parameter(Mandatory = false)]
+        [Parameter(Mandatory = false, HelpMessage = "Overwrites the output file if it exists.")]
```

Comment:
Incorrect help message. Specifying the Force parameter will skip the confirmation question.